src/main/java/frc/robot/commands/CurvatureDriveCommand.java

```diff
@@ -32,10 +32,10 @@ public void execute() {
             double throttle = controls.getLeftY(DEADZONE);
             double turn = controls.getRightX(DEADZONE);
             if(Math.abs(turn) < DEADZONE || Math.abs(throttle) < DEADZONE){
-                drive.arcadeDrive(throttle, turn);
+                drive.arcadeDrive(throttle*.8, turn *.75);
```

Comment:
Add some constants for THROTTLE_PERCENT, TURN_PERCENT, instead of using magic numbers